client/app/components/navbar/navbar.less

```diff
@@ -0,0 +1,48 @@
+.reset-icon() {
+  font-size: inherit;
+  width: auto;
+  height: auto;
+  margin: 0 0 0 rem-calc(5);
+  align-self: center;
+
+  .medium({
+    font-size: rem-calc(10);
+  });
+}
+
+.transition() {
+  display: inline-block;
+  transition: all 0.2s ease-in;
```

Comment:
Using `all` has a cost regarding the animation frame budget.

The only property you have to care about in your case is `transform`.